Engines/Wine/Engine/Implementation/script.js

```diff
@@ -483,7 +483,7 @@ class WineEngine {
     }
 
     changeVersion(containerName) {
-        const wizardTitle = `Change ${containerName} container wine version`;
+        const wizardTitle = tr("Change {0} container wine version", containerName);
```

Comment:
While you're on it anyways: it should be "Wine".

Reply:
Done